### PR TITLE
WIP: drop caddy, step one

### PR DESCRIPTION
**Pull request by kwypchlo**
- drop caddy redirects, only leave caddy for ssl 
- fix cors from nginx
- add route53 support

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -57,6 +61,7 @@ services:
       dockerfile: Dockerfile
     container_name: handshake
     restart: on-failure
+    command: hnd --log-console=false
```
Comment (ro-tex) [inline docker-compose.yml:64]: Shouldn't this be `command: hsd --log-console=false`

File: docker/caddy/Dockerfile [hunk 2]
```diff
@@ -1,20 +1,7 @@
...
-COPY yarn.lock .
-
-ENV CYPRESS_INSTALL_BINARY 0
-RUN yarn --frozen-lockfile
-RUN yarn build
-
 FROM caddy:2.1.1-builder AS caddy-builder
 
-RUN caddy-builder github.com/caddy-dns/cloudflare
+RUN caddy-builder github.com/caddy-dns/cloudflare github.com/caddy-dns/route53
```
Comment (ro-tex) [inline docker/caddy/Dockerfile:3]: Do we still need `github.com/caddy-dns/cloudflare`?

File: docker/nginx/conf.d/client.conf [hunk 3]
```diff
@@ -41,39 +41,102 @@ server {
...
 		proxy_cache skynet;
 		proxy_cache_valid any 1m; # cache stats for 1 minute
+		proxy_set_header User-Agent: Sia-Agent;
 		proxy_pass http://siad-download/skynet/stats;
 	}
 
+	location /health-check {
+		include /etc/nginx/conf.d/include/cors;
+
+		proxy_pass http://health-check:3100;
```
Comment (ro-tex) [inline docker/nginx/conf.d/client.conf:116]: Heathcheck mapped on the same port as Handshake?

Comment (kwypchlo) [reply docker/nginx/conf.d/client.conf:116]: yeah, it's different ip so it's ok

File: docker/caddy/Dockerfile (hunk 2 above)
Comment (kwypchlo) [reply docker/caddy/Dockerfile:3]: not really, leaving it there for anyone else to use for now

File: docker-compose.yml (hunk 1 above)
Comment (kwypchlo) [reply docker-compose.yml:64]: nope, handshake docker container uses `hsd` (handshake daemon) `ENTRYPOINT ["hsd"]`

**Pull request merged**